### ISSUE TITLE
Fix Muon Efficiency Tool

### ISSUE THREAD

**Issue by jdandoy** [jdandoy/BLstopAnalysis#6]
Muon Efficiency Tool implementation is broken, and gives the following error:

```
MuonTriggerScaleFactor... ERROR   /build2/atnight/localbuilds/nightlies/AnalysisBase-2.4.X/AnalysisBase/rel_nightly/MuonEfficiencyCorrections/Root/MuonTriggerScaleFactors.cxx:518 (virtual CP::CorrectionCode CP::MuonTriggerScaleFactors::getMuonEfficiency(Double_t&, const TrigMuonEff::Configuration&, const Muon&, const string&, const string&)): Could not find what you are looking for in the efficiency map. The trigger you are looking for, year and mc are not consistent. Please check how you set up the tool.
MuonEfficiencyCorrecto... WARNING Problem in getTriggerScaleFactor - single muon trigger(s)
MuonTriggerScaleFactor... ERROR   /build2/atnight/localbuilds/nightlies/AnalysisBase-2.4.X/AnalysisBase/rel_nightly/MuonEfficiencyCorrections/Root/MuonTriggerScaleFactors.cxx:518 (virtual CP::CorrectionCode CP::MuonTriggerScaleFactors::getMuonEfficiency(Double_t&, const TrigMuonEff::Configuration&, const Muon&, const string&, const string&)): Could not find what you are looking for in the efficiency map. The trigger you are looking for, year and mc are not consistent. Please check how you set up the tool.
MuonEfficiencyCorrecto... WARNING Problem in getTriggerEfficiency - single muon trigger(s)
```

**Comment by jdandoy** [comment]
The MuonEfficiencyCorrector generally behaves as it should.  See https://twiki.cern.ch/twiki/bin/view/AtlasProtected/MCPAnalysisGuidelinesMC15#Supported_triggers .

Histograms with names such as:
```
_MuonTrigEff_B_HLT_mu26_imedium_OR_HLT_mu50_Loose__EtaPhi_fine_endcap_mc_nominal
_MuonTrigEff_A_HLT_mu26_imedium_OR_HLT_mu50_Loose__EtaPhi_fine_barrel_data_nominal
_MuonTrigEff_A_HLT_mu26_imedium_OR_HLT_mu50_Loose__EtaPhi_fine_barrel_mc_nominal
```
are repeatedly loaded.  At one point, for one event, the following two histograms are loaded:
```
_MuonTrigEff_J_HLT_mu26_imedium_OR_HLT_mu50_Loose__EtaPhi_fine_barrel_data_nominal
_MuonTrigEff_J_HLT_mu26_imedium_OR_HLT_mu50_Loose__EtaPhi_fine_barrel_mc_nominal
```
as these histograms don't exist, the following errors are given:
```
MuonTriggerScaleFactor... ERROR   /home/jdandoy/Documents/SUSY/RPVBLstop/MuonEfficiencyCorrections/Root/MuonTriggerScaleFactors.cxx:520 (virtual CP::CorrectionCode CP::MuonTriggerScaleFactors::getMuonEfficiency(Double_t&, const TrigMuonEff::Configuration&, const Muon&, const string&, const string&)): Could not find what you are looking for in the efficiency map. The trigger you are looking for, year and mc are not consistent. Please check how you set up the tool.
MuonEfficiencyCorrecto... WARNING Problem in getTriggerScaleFactor - single muon trigger(s)
```

**Comment by jdandoy** [comment]
This was fixed in xAODAnaHelpers commit dcb82e7f (https://github.com/UCATLAS/xAODAnaHelpers/pull/676).  Config file has been updated
